twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/ui/profile/ProfileActivity.kt

```diff
@@ -135,18 +135,30 @@ class ProfileActivity : AppCompatActivity(), ProfileContract.View {
     }
 
     override fun showTweets(tweets: MutableList<Status>) {
-        val fragment = TweetsListFragment.newInstance()
-        val args = Bundle()
-
-        // pass the list of tweets to the Tweets List Fragment by argument
-        args.putSerializable(TweetsListFragment.ARG_CODE, tweets as Serializable)
-        fragment.arguments = args
-
-        changeFragment(fragment)
+        hideContent()
+        rv_profile_content_tweets_list.visibility = View.VISIBLE
+
+        rv_profile_content_tweets_list.apply {
```

Comment:
Can you heck if the same comment about the `rv_main_trends_list.apply { ... }` on the _MainActivity_ applies to this case.